docs/course_materials/linux_intro.html

```diff
@@ -0,0 +1,367 @@
+          </ul>
+          <pre><code data-trim data-noescape class="hljs">
+$pwd
+/home/student
+$
+        </code></pre>
+        </section>
+      </section>
+      <section>
+        <section>
+          <h3><b>mkdir</b></h3>
+          <ul>
+            <li>Creates a directory in the current directory</li>
+          </ul>
+          <pre><code data-trim data-noescape class="hljs">
+$mkdir my_test
+$
+        </code></pre>
+          <ul>
+            <li>Creates a directory in a specific directory</li>
+          </ul>
+          <pre><code data-trim data-noescape class="hljs">
+$mkdir /home/student/my_test
+$
+        </code></pre>
+          <ul>
+            <li>Creates a directory and all parent directories if missing</li>
+          </ul>
+          <pre><code data-trim data-noescape class="hljs">
+$mkdir -p my_test/dir2/dir3/dir4
+$
+        </code></pre>
+        </section>
+      </section>
+      <section>
+        <section>
+          <h3><b>cd</b></h3>
+          <ul>
+            <li>Change to another directory existing in the current directory</li>
+          </ul>
+          <pre><code data-trim data-noescape class="hljs">
+$cd my_test
+$
+        </code></pre>
+          <ul>
+            <li>Change to a specific directory (Full path)</li>
+          </ul>
+          <pre><code data-trim data-noescape class="hljs">
+$cd /home/student/my_test
+$
+        </code></pre>
+        </section>
+        <section>
+          <h3><b>cd</b></h3>
+          <ul>
+            <li>Change to user home directory</li>
+          </ul>
+          <pre><code data-trim data-noescape class="hljs">
+  $cd
+  $
+          </code></pre>
+          <pre><code data-trim data-noescape class="hljs">
+  $cd ~
+  $
+          </code></pre>
+          <ul>
+            <li>Change to parent directory</li>
+          </ul>
+          <pre><code data-trim data-noescape class="hljs">
+  $cd ..
+  $
+          </code></pre>
+        </section>
+      </section>
+      <section>
+        <section>
+          <h3><b>ls</b></h3>
+          <ul>
+            <li>List current directory content</li>
+          </ul>
+          <pre><code data-trim data-noescape class="hljs">
+  $ls
+  my_test
+  $
+          </code></pre>
+          <ul>
+            <li>List specific directory content</li>
+          </ul>
+          <pre><code data-trim data-noescape class="hljs">
+  $ls /home/student
+  my_test
+  $
+          </code></pre>
+        </section>
+        <section>
+          <h3><b>ls -la</b></h3>
+          <ul>
+            <li>List current directory content with additional information and hidden files</li>
+          </ul>
+          <pre><code data-trim data-noescape class="hljs">
+  $ls -la
+  total 12K
+  drwxrwxr-x.  3 student student 4.0K Dec  9 22:57 .
+  drwxrwxr-x. 10 student student 4.0K Dec  9 22:45 ..
+  -rw-rw-r--.  1 student student    0 Dec  9 22:57 .bashrc
+  drwxrwxr-x.  2 student student 4.0K Dec  9 22:45 my_test
```

Comment:
same as 176, although maybe ls -la is "too much"?

Reply:
Done